python/ray/WebUI.ipynb

```diff
@@ -66,20 +199,76 @@
    },
    "outputs": [],
    "source": [
-    "task_search = widgets.Text(\n",
-    "    value=\"\",\n",
-    "    placeholder=\"Task ID\",\n",
-    "    description=\"Search for a task:\",\n",
-    "    disabled=False\n",
-    ")\n",
-    "display(task_search)\n",
+    "def task_timeline():\n",
+    "    path_input = widgets.Button(description=\"View task timeline\")\n",
+    "\n",
+    "    breakdown_basic = \"Basic\"\n",
+    "    breakdown_task = \"Task Breakdowns\"\n",
+    "    \n",
+    "    breakdown_opt = widgets.Dropdown(\n",
+    "        options=[\"Basic\", \"Task Breakdowns\"],\n",
+    "        value=\"Basic\",\n",
+    "        description=\"View options:\",\n",
+    "        disabled=False,\n",
+    "    )\n",
+    "    \n",
+    "    start_box, end_box, range_slider, time_opt = get_sliders(False)\n",
+    "    display(breakdown_opt)\n",
+    "    display(path_input)\n",
+    "\n",
+    "    def find_trace2html():\n",
+    "        trace2html = os.path.join(os.path.expanduser(\"~\"), \"catapult\", \"tracing\", \"bin\", \"trace2html\")\n",
+    "        assert os.path.exists(trace2html), \"Could not find catapult, please clone it into your home directory from https://github.com/catapult-project/catapult/tree/master/tracing\"\n",
```

Comment:
This is going to be problematic. The right thing to do is probably to install it when we build Ray. However, for now we can just clone it right here when this function runs (somewhere under `/tmp/`) if it isn't already there.